.github/workflows/tests.yaml

```diff
@@ -0,0 +1,34 @@
+name: tests
+on:
+  push:
+  pull_request:
+jobs:
+  OS:
```

Comment:
"OS"?

Reply:
"operating system", as it's a matrix. I always struggle with naming a single job in an already named "tests" workflow. Any preference? "unit" might work, it will be shown as "tests / unit" in the PR then. Or just "run"?